event_routing_backends/models.py

```diff
@@ -40,6 +43,31 @@ def get_value_from_dotted_path(dict_obj, dotted_key):
     return result
 
 
+class RouterConfigurationManager(ConfigurationModelManager):
+    """
+    Query manager for ConfigurationModel.
+    """
+
+    def get_routers(self, backend_name):
+        """
+        Bring active routers of a backend.
+
+        A queryset for the active configuration entries only. Only useful if backend_name is passed.
+        This function will return all active routers of a backend.
+        """
+        if not backend_name:
+            return []
+
+        cache_key = get_cache_key(site_domain="example.com", resource=backend_name)
```

Comment:
we don't need to pass `site_domain` here. instead we can pass `namespace` like this `namespace="event_routing_backends"`